docker/db.Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM postgres:latest
+
+COPY init-user-db.sh /usr/local/bin
+RUN ln -s /usr/local/bin/init-user-db.sh /docker-entrypoint-initdb.d/init-user-db.sh
+
+CMD ["postgres"]
```

Comment:
Maybe use the  basic `postgres` image here instead of defining a new docker image. The configuration you make in the `docker/init-user-db.sh` file is included in the postgres docker image (https://hub.docker.com/_/postgres)